### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by gneatgeek**
* Bumped the version number
* Updated the `lint-scss` script
* Updated the release process

### CODE REVIEW

**Comment by windse7en** [comment]
+1

File: RELEASE.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ This project is hosted on NPM.  You can see it [here][project-url].
 
 Releasing the project requires these steps:
 
-0. Set the version number in the code
-1. `npm run webpack`
+0. Set the version number in `package.json`
+1. Check for lint: `npm run lint`
 2. Use a GitHub [project release][github-release-url] to release the project and tag (be sure it follows [semver][semantic-versioning])
 3. `npm publish` (make sure it's uploaded to [npm][project-url])
 4. Update `master` to a new minor version
```
Comment (Matt-Butler) [inline RELEASE.md]: I think step 4 needs to be updated too.  

```
4. Update to next minor version in `package.json`.
```

Comment (Matt-Butler) [inline RELEASE.md]: Actually, step 1 isn't needed either

**Comment by Matt-Butler** [comment]
+1

**Pull request merged**